fermipy/defaults.py

```diff
@@ -101,7 +101,6 @@ def make_default_dict(d):
     'resample': (True, '', bool),
     'srcmap': (None, '', str),
     'bexpmap': (None, '', str),
-    'wmap': (None, 'Likelihood weights map.', str),
```

Comment:
It looks like this commit is reverting recent changes in gtanalysis and defaults.  We'll need to fix this before this can be merged.